test/clientTest.js

```diff
@@ -41,31 +38,31 @@ mc.supportedVersions.forEach(function (supportedVersion, i) {
       })
     })
 
-    describe('offline', function () {
+    describe('Offline mode tests', function () {
       before(function (done) {
-        console.log(new Date() + 'starting server ' + version.minecraftVersion)
+        console.log(`      Starting minecraft server for ${version.minecraftVersion}`)
         wrap.startServer({
           'online-mode': 'false',
           'server-port': PORT,
-          'motd': 'test1234',
+          'motd': MOTD,
           'max-players': 120
         }, function (err) {
           if (err) { console.log(err) }
-          console.log(new Date() + 'started server ' + version.minecraftVersion)
+          console.log(`      Started minecraft server for ${version.minecraftVersion}`)
```

Comment:
Leave the date

Reply:
If you want to know the startup time, then extract it from the server log and print it with the server started text.